vocode/streaming/streaming_conversation.py

```diff
@@ -1378,6 +1378,11 @@ def is_interrupt(self, transcription: Transcription):
             self.transcriber.get_transcriber_config().min_interrupt_confidence or 0
         )
 
+    def get_duration_spoken_seconds(self, time_started_speaking: float | None):
+        if time_started_speaking is None:
+            return 0
```

Comment:
Right now, time_started_speaking is always a float from where we call this function. In future prs it can be None.